src/app/common-ui/Popup/index.js

```diff
@@ -17,29 +26,24 @@ function Popup({ show, title, content, buttonText, variant, onButtonClick }) {
     }
   }, [show]);
 
+  const selectSign = (variant) => {
+    if (variant === POPUP_VARIANT.ERROR) return iconError;
+    if (variant === POPUP_VARIANT.SUCESS) return iconSuccess;
+    if (variant === POPUP_VARIANT.WARNING) return iconWarning;
+    return '';
+  };
+
   return !isPristine ? (
     ReactDOM.createPortal(
       <div className={cx(show && styles.overlay)}>
         <div className={cx(styles.container, !show ? styles.close : styles.open)}>
-          <div className={styles.titleBar}>
-            <div
-              className={cx(
-                styles.title,
-                variant === 'success' ? styles.titleColorSuccess : styles.titleColorFailed,
-              )}
-            >
-              {title}
-            </div>
-            <div className={styles.closeImg} onClick={onButtonClick}>
-              <img src={CloseButton} alt="close" />
-            </div>
+          <div className={styles.closeImg} onClick={onButtonClick}>
+            <img src={CloseButton} alt="close" />
           </div>
-          <div className={styles.content}>
-            <div>{content}</div>
+          <div className={styles.iconContainer}>
+            <img className={styles.sign} src={selectSign(variant)} alt="popupSign" />
```

Comment:
rename `styles.sign` to `styles.icon`
remove `alt` unless you change it to something which is meaningful to blind users